### PR TITLE
Add test dependancies

### PR DESCRIPTION
**Pull request by seanwiseman**

### CODE REVIEW

**Comment by seanwiseman** [comment]
Ok, that's done for this PR, just allows the `project_tests.sh` to be run (linting and tests) and tidied up the structure.

File: app/core/urls.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+from django.contrib import admin
+from django.urls import path
+
+URLPATTERNS = [
+    path('admin/', admin.site.urls),
```
Comment (giorgiosironi) [inline app/core/urls.py]: is this URL required? Can't think of a use case

File: app/project_tests.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -e
+
+pipenv install --dev
```
Comment (giorgiosironi) [inline app/project_tests.sh]: this should be running in the container so it should already have a virtual environment and the dependencies installed

File: app/core/urls.py (hunk 1 above)
Comment (seanwiseman) [reply app/core/urls.py]: No, it can go. I will replace with a `ping` endpoint so there is something to hit.

File: app/project_tests.sh (hunk 2 above)
Comment (seanwiseman) [reply app/project_tests.sh]: Ah, `${pipenv_dev_arg}` is taking care of it!

File: app/core/settings.py [hunk 3]
```diff
@@ -3,7 +3,7 @@
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 SECRET_KEY = 'bi6y(ew+jh54zu-gs#yete^c13_8f##jgq3lj9s&nfg-6!06zn'
 DEBUG = True
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = '*'
```
Comment (seanwiseman) [inline app/core/settings.py]: Will be replaced with config.

**Pull request merged**